flink-table/flink-table-common/src/main/java/org/apache/flink/table/typeutils/TimeIndicatorTypeInfo.java

```diff
@@ -61,10 +62,13 @@ protected TimeIndicatorTypeInfo(boolean isEventTime) {
         this.isEventTime = isEventTime;
     }
 
+    // this replaces the effective serializer by a LongSerializer
+    // it is a hacky but efficient solution to keep the object creation overhead low but still
+    // be compatible with the corresponding SqlTimestampTypeInfo
     @Override
     @SuppressWarnings("unchecked")
     public TypeSerializer<Timestamp> createSerializer(ExecutionConfig executionConfig) {
-        return (TypeSerializer) LocalDateTimeSerializer.INSTANCE;
+        return (TypeSerializer) LongSerializer.INSTANCE;
```

Comment:
Any tests releated to this change?

Reply:
They modify the `TimeIndicatorTypeInfo` is because it will break the test `FunctionITCase#testAggregateFunction`. Now I revert the change in the `TimeIndicatorTypeInfo` and make sure the test still passes. So I think the origin test is enough.

Reply:
ok